extensions/amp-analytics/0.1/requests.js

```diff
@@ -250,6 +289,7 @@ export class RequestHandler {
     this.queueSize_ = 0;
     this.baseUrlPromise_ = null;
     this.baseUrlTemplatePromise_ = null;
+    this.requestOriginPromise_ = null;
```

Comment:
I couldn't come up with a good use case where we want to reset `requestOriginPromise` when we reset the batching status. 
What do you think? Should we support `requestOrigin` just like `baseUrl`, or do we want to be more restrictive on this specific usage. I am a bit concerned and want to confirm the default behavior. Because it would be a breaking change if we want to make a change later.

Reply:
Discussed offline, we will not support dynamic `requestOrigin` unless someone asks for it, so we will not reset `requestOriginPromise_`